packages/vscode-ext/package.json

```diff
@@ -115,24 +115,25 @@
   },
   "scripts": {
     "vscode:prepublish": "yarn run compile",
-    "compile:tsc": "tsc -b",
```

Comment:
I think this is still necessary and shouldn't be removed. Maybe not in the compile target but in `lint` or in `pretest`?

Reply:
Lint is just an option, I'm not sure when this should be done. But we need to type-check the typescript code _somewhere_ in the build process.

Reply:
You are right on typechecking, yeah. Added `yarn typecheck` (which calls tsc in noEmit mode), and made it a mandatory step in prepublish

Reply:
I thought about this a bit more and I think `typecheck` (or whatever you call it) should be called from one of `build`, `test`, or `lint` steps, not just prepublish. I don't really care from which one, but build/test/lint are the only steps I'm going to call from the new turborepo-powered CI, in most cases.

(Btw, I'll also rename `compile` to `build` for this package for that reason in my upcoming PR).

Reply:
Sure, lint it is)